models/honeyswap/gnosis/honeyswap_v2_gnosis_schema.yml

```diff
@@ -0,0 +1,73 @@
+version: 2
+
+models:
+  - name: honeyswap_v2_gnosis_trades
+    meta:
+      blockchain: gnosis
+      sector: dex
+      project: honeyswap_v2
+      contributors: 0xr3x, jeff-dude, masquot, soispoke
+    config:
+      tags: ['gnosis','honeyswap_v2','trades', 'dex']
+    description: >
+        Honeyswap V2 contract trades on Gnosis
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+      - check_dex_seed:
+          blockchain: gnosis
+          project: honeyswap_v2
+          version: 2
```

Comment:
Hi @m-r-g-t 
in order for the `check_dex_seed` test to work,
we can add some seeds in this file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv

Reply:
@Hosuke could you explain a bit on what this does? Is it just a matter of adding a couple transactions for testing purposes?

Reply:
added in https://github.com/duneanalytics/spellbook/pull/4139/commits/ccc7a8c3d9130836fe0d16f8247a27b063f3b85d

Reply:
The test will compare query results with the records in this seed file.